### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by bam95899**
Lots of refactoring & adding in a profile page

### CODE REVIEW

File: sciencelabs/templates/nav.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
     {% set user = "Jordan Jasik" %}
 
-    <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #0069AA;">
+    <nav class="navbar navbar-expand-lg navbar-dark" style="padding: 0px 10px; background-color: #0069AA;">
```
Comment (c-schwarze) [inline sciencelabs/templates/nav.html]: inline styles should be pulled out to .css file

File: sciencelabs/templates/nav.html [hunk 2]
```diff
@@ -12,58 +12,36 @@
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
             <ul class="navbar-nav mr-auto">
                 <li class="nav-item">
-                    <a id="session" class="nav-link" href="{{ url_for('SessionView:index') }}" style="color:#FFC600;">Session</a>
+                    <a id="session" class="nav-link {{ 'nav-bar-active' if 'SessionView' in request.url_rule.endpoint else 'disabled' }}" href="{{ url_for('SessionView:index') }}" style="color:#FFC600;">Session</a>
```
Comment (c-schwarze) [inline sciencelabs/templates/nav.html]: You probably could remove the else and just not output a class?

Comment (c-schwarze) [reply sciencelabs/templates/nav.html]: It looks slick though 👍

File: sciencelabs/templates/reports/monthly.html [hunk 3]
```diff
@@ -39,25 +39,25 @@
...
-                        <p><strong>Session Statistics for ***MONTH***</strong></p>
+                        <h4>Session Statistics for ***MONTH***</h4>
                     </div>
                 </div>
                 <div class="form-row">
                     <div class="form-group col-md-12">
-                     <label for="session-name">Search:</label>
-                    <input type="text" class="form-control" id="session-name">
+                     <label for="search">Search:</label>
+                    <input type="text" class="form-control" id="search">
```
Comment (c-schwarze) [inline sciencelabs/templates/reports/monthly.html]: lines 47/48 are identical to 59/60. Both have id="search" .

File: sciencelabs/templates/nav.html (hunk 2 above)
Comment (bam95899) [reply sciencelabs/templates/nav.html]: Removed else.

File: sciencelabs/templates/nav.html (hunk 1 above)
Comment (bam95899) [reply sciencelabs/templates/nav.html]: Fixed

**Pull request merged**